doc/user/content/sql/functions/to_char.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "to_char Function"
+description: "Converts a timestamp into a string using the specified format."
+menu:
+  main:
+    parent: 'sql-functions'
+---
+
+`to_char` returns a string created using the specified format string.
+
+The format string can be composed of any number of format tokens, interspersed with
+regular text. You can place a format token inside of double-quotes to emit it literally.
+
+## Examples
+
+**RFC 2822 Format**
```

Comment:
This seems like a heading; it should probably just be a heading? Also should be sentence case.

Reply:
Can do, it just looks weird using this as an h3.

Reply:
For reference, with just the bold:

<img width="374" alt="image" src="https://user-images.githubusercontent.com/277161/100776775-f9153f00-33d2-11eb-8baf-57ffc0bf7b9b.png">

I'll update this comment with screenshots of the other headings. I think I tried down to h4 and they all looked weird to me, but we can look.

Reply:
So it seems to me that the heading styling needs tweaking if we're abandoning semantics for the sake of formatting.  🤷 not worth holding up this PR over at all!

Reply:
I agree, I opted for semantics over style, we can change it later.